tests/checkpoint/test_checkpoint.py

```diff
@@ -265,649 +217,6 @@ def test_basic_checkpoint_config_validation(
     assert len(context.list_checkpoints()) == 0
 
 
-@pytest.mark.filesystem
-def test_checkpoint_configuration_no_nesting_using_test_yaml_config(
```

Comment:
Deleting any tests around `test_yaml_config` since we're removing it from the codebase as part of V1